buildDockerImages.pl

```diff
@@ -229,10 +230,22 @@ sub cleanupAfterBuild {
 my $logFile = "buildDockerImages.log";
 open( $fileout, ">$logFile" ) or die "Can't open file $logFile for writing: $!\n";
 
+my $version = `cat version.txt`;
+chomp($version);
+
 # Build the executables
 print "Building the executables.\n";
 print $fileout "Building the executables.\n";
-runAndLog($fileout, "./gradlew release");
+
+# Create a .gradle directory and map it into the container
+# This will speed susequent builds
+my $cwd = getcwd();
+`mkdir -p $cwd/.gradle`;
+my $cmdString = "docker run --name weathervane-builder --rm "
+              . "-v $cwd/.gradle:/root/.gradle " 
+              . "-v $cwd:/root/weathervane -w /root/weathervane " 
+              . "--entrypoint /root/weathervane/gradlew openjdk:8 release";
```

Comment:
I have a basic question here. The docker run command, that I can see, doesn't specify an image. Does it use a default image then?

Reply:
I see, thank you.